nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -90,7 +90,12 @@ class BtActionNode : public BT::ActionNodeBase
 
     // Make sure the server is actually there before continuing
     RCLCPP_DEBUG(node_->get_logger(), "Waiting for \"%s\" action server", action_name.c_str());
-    action_client_->wait_for_action_server();
+    if (!action_client_->wait_for_action_server(server_timeout_)) {
```

Comment:
Ok I think I have to rethink this a small bit. The default `server_timeout` is 20ms, which is not a lot of time to wait by default. 
Should I declare a new parameter for this timeout?

Reply:
I'm hesitant to have a new timeout since that might confuse people, even if well named - but maybe there's a good name that would be clear so its moot? `wait_for_action_server_timeout` `server_initialization_timeout`?

Otherwise, if we can't come up with a good & specific name, I think just hardcoding it at some value would be fine. I'd say 1s just to be nice and round. We have a few like that (only a few) in the stack where it didn't make much sense to parameterize it for various reasons and since they were just initialization operations, it wasn't super important for them to be configurable for 99.8% of users.